File: MS/src/main/java/com/yskc/ms/entity/ms/AppMenu.java
package com.yskc.ms.entity.ms;

import com.baomidou.mybatisplus.annotations.TableId;
import com.baomidou.mybatisplus.annotations.TableName;

import java.io.Serializable;

/**
 * 系统菜单
 * @author huronghua
 */
@TableName("app_menu")
public class AppMenu  implements Serializable {
    private static final long serialVersionUID = 1L;
    @TableId
    private Integer id;
    private Integer parentId;
    private String name;
    private int type;
    private int level;
    private String url;
    private String icon;
    private String perms;
    private int status;
    private String seq;
    private String remark;
    private String fullPath;
    private Integer platform;

    public Integer getPlatform() {
        return platform;
    }

    public void setPlatform(Integer platform) {
        this.platform = platform;
    }

    public String getFullPath() {
		return fullPath;
	}

	public void setFullPath(String fullPath) {
		this.fullPath = fullPath;
	}

	public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getParentId() {
        return parentId;
    }

    public void setParentId(Integer parentId) {
        this.parentId = parentId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }

    public int getLevel() {
        return level;
    }

    public void setLevel(int level) {
        this.level = level;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getIcon() {
        return icon;
    }

    public void setIcon(String icon) {
        this.icon = icon;
    }

    public String getPerms() {
        return perms;
    }

    public void setPerms(String perms) {
        this.perms = perms;
    }

    public int getStatus() {
        return status;
    }

    public void setStatus(int status) {
        this.status = status;
    }

    public String getSeq() {
        return seq;
    }

    public void setSeq(String seq) {
        this.seq = seq;
    }

    public String getRemark() {
        return remark;
    }

    public void setRemark(String remark) {
        this.remark = remark;
    }
}

File: YSDB/MSDB/Models/app_menu.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace MSDB.Models
{
    /// <summary>
    /// 功能菜单表
    /// </summary>
    public class app_menu
    {
        [Key]
        public int id { get; set; }

        public int parentId { get; set; }

        /// <summary>
        /// 功能菜单名
        /// </summary>
        [Required, MaxLength(30)]
        public string name { get; set; }

        /// <summary>
        /// 类型，0:菜单 1：目录 2：按钮
        /// </summary>
        public byte type { get; set; }

        /// <summary>
        /// 级别
        /// </summary>
        public byte level { get; set; }

        /// <summary>
        /// 菜单地址
        /// </summary>
        [MaxLength(200)]
        public string url { get; set; }

        /// <summary>
        /// 图标
        /// </summary>
        [Required, MaxLength(30)]
        public string icon { get; set; }

        /// <summary>
        /// 权限标识，形如：user:list,user:create
        /// </summary>
        [MaxLength(200)]
        public string perms { get; set; }

        /// <summary>
        /// 功能状态，0：不可见，1：可见
        /// </summary>
        public byte status { get; set; }

        /// <summary>
        /// 排序
        /// </summary>
        public byte seq { get; set; }

        /// <summary>
        /// 备注
        /// </summary>
        [MaxLength(300)]
        public string remark { get; set; }

        /// <summary>
        /// 全路径
        /// </summary>
        [MaxLength(300)]
        public string fullPath { get; set; }

        ///</summary>
        ///平台：0：PC，1：移动
        ///</summary>
        public int platform { get; set; }
    }
}
